### ISSUE TITLE
arstechnica.com

### ISSUE THREAD

**Issue by deusexcalamus** [easylist/easylist#4141]
### List the website(s) you're having issues:
arstechnica.com articles, ex: https://arstechnica.com/tech-policy/2019/10/senator-proposes-mandatory-labeling-for-products-with-mics-cameras/

### What happens?
Unblocked sponsor content at end of article

### List Subscriptions you're using:
all builtin filters except experimental
Adblock Warning Removal List
EasyList
EasyPrivacy
Fanboy's Enhanced Tracking List
Malware Domain List
Malware domains
Peter Lowe's Ad and tracking server list
Fanboy's Cookie List
NoCoin Filter List

### Your settings
had HTTPS Anywhere enabled, disabled it, same result.

- OS/version: Win10/Android10
- Browser/version: 69.0.2 (on w10), 68.2b6 on Android
- Adblock Extension/version: 1.22.4 on both

### Other details:
Long story short, this was fixed in e7762d813b73ebf943423ac125f853850337f415 but seems to have been removed in the interim.

**Comment by Khrin** [comment]
In the past, has been decided to consider Actionbutton stuff as annoyance, rather as advertising. The blocking rule is in the Annoyances List: https://github.com/easylist/easylist/blob/master/fanboy-addon/fanboy_annoyance_thirdparty.txt#L10

**Comment by deusexcalamus** [comment]
ty 👍
